alf/trainers/policy_trainer.py

```diff
@@ -663,18 +679,33 @@ def play(root_dir,
             buffer_size=num_episodes, reward_shape=env.reward_spec().shape),
         alf.metrics.AverageEpisodeLengthMetric(buffer_size=num_episodes),
     ]
+
     while episodes < num_episodes:
-        time_step, policy_state, trans_state, info = _step(
+
+        # transform time step
+        transformed_time_step, trans_state = algorithm.transform_timestep(
```

Comment:
_step will do transform_step inside. Do you really need transformed_time_step to be recorded?